source/release-notes.txt

```diff
@@ -10,6 +10,16 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.x
+----------------
+
+*Upcoming*
+
+New Features
+
+- Queries generated by Natural Language Querying have improved 
+  quality and accuracy.
```

Comment:
We are planning to make some UX improvements, that will probably can require docs changes. Will we include those in this PR? Should we maybe do a review closer to the expected release day when we have more info?

Reply:
Hi @alenakhineika, I am fine with leaving the screenshots for another ticket if required. Its more important that the feature is clearly marked as production ready for .local NYC next week. Please file an additional ticket if the GUI changes drastically and I can treat as a fast follow. Ideally we would have the docs fully reviewed this week and can hold the publish to coincide with the release. @jaoppenheim